### DISCUSSION TITLE
Longer timer duration options

### DISCUSSION THREAD

**Discussion by jordanh** [ParabolInc/parabol#5741] [Feedback]
[A user](https://app.hubspot.com/contacts/3888472/contact/38441701) 🔒 writes: 

> The timer is also hard to use because it puts the burden on the facilitator, who is often herding cats, to remember to start it. Forgetting to do things (such as taking notes or starting recording etc) is a real problem during meetings. (Being a small icon at the bottom doesn't help with this.) With LCT I'm prompted to pick a period for the timer to run before each topic, so I don't have to remember that. 
>
> Even if I remember to use it, the 9 minute maximum is not enough and really needs more options like 10-15-20-30 minutes or at least the "till this time" option should have smaller increments than 30 minutes.

**Comment by jordanh** [comment]
[A user](https://app.hubspot.com/contacts/3888472/contact/3861651) 🔒 writes:

> It would be great if you could put more than 9 minutes on the clock, best if you could just enter number you wanted.

**Comment by jordanh** [comment]
@enriquesanchez as you'd been looking at picking time durations with prompt updates, I wanted to mention you for this one as they are kinda in the same neighborhood

**Comment by gcrickman** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/11070051) 🔒  mentioned:

> We would like to be able to set the timer for longer than 9 minutes.

**Comment by enriquesanchez** [comment]
Instead of a select field with 9 options (1 to 9 minutes), we could just have a text field as suggested above.

![image](https://user-images.githubusercontent.com/3276087/144326122-58180671-3ee6-4256-aeb3-654ed12efaf6.png)

The field could be pre-filled with a default, such as `5`, and then users can change the value as needed.
